crates/services/importer/src/importer/test.rs

```diff
@@ -236,43 +220,37 @@ where
 #[test_case(
     genesis(0),
     underlying_db(not_found),
-    executor_db(ok(0), ok(None), ok(None), 1)
+    executor_db(ok(0), ok(None), 1)
```

Comment:
test_case's should be documented with human readable descriptions, ie.

`#[test_case(...; "successfully imports genesis block when database is empty")`

Reply:
created issue #940 to address this as a followup.